tools/bin/hawqregister

```diff
@@ -729,6 +728,23 @@ class HawqRegister(object):
                 for k, eof in enumerate(eofs[1:]):
                     query += ',(%d, %d, %d, %d, %d)' % (self.firstsegno + k + 1, eof, -1, -1, -1)
```

Comment:
Thanks @kdunn-pivotal for the good advice. But I think upgrade from Python2.x to Python3.x is a big deal and since we are not familiar with Python3.x, current code is ok. For the syntax you mentioned here, you are right. TIL